### PR TITLE
fixed Not able to update decimal quantity from cart details page. #24509

### PR DESCRIPTION
**Pull request by rani-priya**
### Fixed Issues (if relevant)

1. magento/magento2#24509: fixed Not able to update decimal quantity from cart details page.

### Manual testing scenarios (*)

1. create a product
2. set "Qty uses decimal" to yes
3. add product to cart
4. navigate to cart page and try to update decimal qty

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by rogyar** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rogyar. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rogyar, here is your [new Magento instance](https://pr-24510.instances.magento-community.engineering).
Admin access: [https://pr-24510.instances.magento-community.engineering/admin](https://pr-24510.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml [hunk 1]
```diff
@@ -50,7 +50,7 @@ $canApplyMsrp = $helper->isShowBeforeOrderConfirm($product) && $helper->isMinima
                                 <?php if (isset($_formatedOptionValue['full_view'])) :?>
                                     <?= $block->escapeHtml($_formatedOptionValue['full_view']) ?>
                                 <?php else :?>
-                                    <?= $block->escapeHtml($_formatedOptionValue['value'], ['span', 'a']) ?>
+                                    <?= $block->escapeHtml($_formatedOptionValue['value'], ['span']) ?>
```
Comment (rogyar) [inline app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml]: Is this change also related to the current bugfix? Just want to clarify the reason for changing item options rendering upon fixing the Qty input behavior.

Thank you.

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml (hunk 1 above)
Comment (rani-priya) [reply app/code/Magento/Checkout/view/frontend/templates/cart/item/default.phtml]: No.. I have reverted it.

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-5811](https://jira.corp.magento.com/browse/ENGCOM-5811) has been created to process this Pull Request

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-5811](https://jira.corp.magento.com/browse/ENGCOM-5811) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed
